splash/qtrender_lua.py

```diff
@@ -1095,8 +1168,11 @@ def get_wrapped(self):
 
     def run_async_command(self, cmd):
         """ Execute _AsyncBrowserCommand """
-        meth = getattr(self.tab, cmd.name)
-        return meth(**cmd.kwargs)
+        if isinstance(cmd, AsyncBrowserCommand):
+            meth = getattr(self.tab, cmd.name)
+            return meth(**cmd.kwargs)
+        elif isinstance(cmd, AsyncLuaCommand):
+            cmd.kwargs["callback"]()
```

Comment:
What do you think about renaming it to AsyncCallbackCommand? For AsyncLuaCommand I'd expect callback to be a Lua callback, but here it is an arbitrary Python function